monitoring/my-kube-prometheus/unchained-kube-promstack.jsonnet

```diff
@@ -0,0 +1,93 @@
+local addArgs = (import './utils.libsonnet').addArgs;
+
+local kp =
+  (import 'kube-prometheus/main.libsonnet') +
+  (import 'kube-prometheus/addons/networkpolicies-disabled.libsonnet') +
+  {
+    values+:: {
+      common+: {
+        namespace: 'unchained-monitoring',
+      },
+      alertmanager+: {
+       config: importstr 'alertmanager-config.yaml',
+      },
+      prometheus+:: {
+        // The namespaces to monitor with Prometheus.
+        namespaces+: ['unchained-dev'],
+      },
+      grafana+:: {
+        config+: {
+          sections+: {
+            server+: {
+              root_url: 'http://grafana.dev.unchained.com/',
```

Comment:
not a dns we own. `monitoring.shapeshift.com`?

Reply:
done